### ISSUE TITLE
Allow/disallow curly braces for string literal attributes (enforce double/single quotes only)

### ISSUE THREAD

**Issue by spradlin-dev** [palantir/tslint-react#246]
existing quotes rule `jsx-double` only makes sure you don't type `attr='value'`, but doesn't prevent `attr={'value'}`.  

I want enable my team to catch inconsistent quotes in JSX and only use curly braces for string values when they have to be interpreted or concatenated, such as using string templates.

### No error
```jsx
<Component name="string"/>
<Component name={'string-' + id}/>
<Component name={`string-${id}`}/>
```

### Error
```jsx
<Component name={'string'}/>
<Component name={"string"}/>
```

**Comment by adidahiya** [comment]
Sounds like a good idea, but I would suggest making this an ESLint rule instead. See https://github.com/palantir/tslint-react/issues/210
